sklearn/feature_extraction/text.py

```diff
@@ -681,7 +688,7 @@ def fit(self, X, y=None):
 
         self._validate_params()
 
-        self._get_hasher().fit(X, y=y)
+        self.hasher = self._get_hasher().fit(X, y=y)
         return self
```

Comment:
The docstring about this estimator being stateless need to change if we decide to go that way.